Use lua instead of luaJIT by default (#916)
apache/incubator-kvrocks: README.md

```shell
$ ./x.py build -DENABLE_OPENSSL=ON
```

> Recommended for higher performance optimization
> * Lua
>   * Use luajit instead of lua

Reviewer: ```suggestion
To build with luaJIT instead of lua for better performance, run:
```

  ```shell
  ./x.py build -D USE_LUAJIT=ON

>   ```

Reviewer: delete these indent

$ ./x.py build -DENABLE_OPENSSL=ON
```

To build with luaJIT instead of lua for better performance, run:

```shell

> $ ./x.py build -D USE_LUAJIT=ON

Reviewer: ```suggestion
$ ./x.py build -DUSE_LUAJIT=ON
```

  * and most other distros
* macOS

### Recommended for higher performance optimization
* Lua
  * Use luajit instead of lua
  ```shell
  ./x.py build -D USE_LUAJIT=ON
  ```

> 

Reviewer: You can add this section under the "To build with TLS support" section.
